website/blog/2023-12-04-0.73-debugging-improvements-stable-symlinks.md

```diff
@@ -0,0 +1,178 @@
+
+:::info
+Remote JavaScript Debugging was previously the default debugging experience for apps using JavaScriptCore (JSC). We recommend [Safari Developer Tools (direct JSC debugging)](/docs/next/other-debugging-methods#safari-developer-tools-direct-jsc-debugging) instead, for iOS apps.
+
+We recommend using [Hermes](/docs/hermes) for a consistent debugging experience on all platforms.
+:::
+
+### New Architecture Updates
+
+We continue active development on the New Architecture and are fully invested in bringing it to the OSS community. You can read the latest updates in the [New Architecture working group](https://github.com/reactwg/react-native-new-architecture/discussions).
+
+## Breaking Changes
+
+### Babel Package Renames
+
+We've relocated two Babel-related packages out of Metro and into React Native's repository and versioning scheme. The new versions of these packages support New Architecture features in 0.73, meaning these dependencies must be updated.
+
+Please follow the [upgrade helper](https://react-native-community.github.io/upgrade-helper/) when upgrading, to ensure you have updated these dependencies. If you were using `metro-react-native-babel-preset` in your Babel config file, don't forget to update that too.
+
+There are no changes to the [react-native](https://www.npmjs.com/package/react-native) package.
+
+| Old Package Name                       | New Package Name                        |
+| -------------------------------------- | --------------------------------------- |
+| `metro-react-native-babel-preset`      | `@react-native/babel-preset`            |
+| `metro-react-native-babel-transformer` | `@react-native/metro-babel-transformer` |
+
+:::info
+`@react-native/babel-preset` now includes `@react-native/babel-plugin-codegen`, this no longer needs to be specified separately in your Babel config file.
+:::
+
+### React Native CLI Changes
+
+#### Highlighted breaking changes
+
+- Change default `taskPrefix` in build-android command from `assemble` to `bundle` ([#1913](https://github.com/react-native-community/cli/pull/1913))
+- Remove fallback flow for Metro config defaults ([#1972](https://github.com/react-native-community/cli/pull/1972))
+- Remove `--configuration` option from `run-ios` (replaced with `--mode`) ([#2028](https://github.com/react-native-community/cli/pull/2028))
+- Remove `--variant` option from `build-android` command (replaced with `--mode`) ([#2026](https://github.com/react-native-community/cli/pull/2026)).
+
+[See full changelog for v12.0.0](https://github.com/react-native-community/cli/releases/tag/v12.0.0).
+
+### Other Breaking Changes
+
+- Raise minimum Node.js requirement to 18.x ([#37709](https://github.com/facebook/react-native/pull/37709))
+- Major bump of TypeScript to 5.0 ([#36862](https://github.com/facebook/react-native/pull/36862))
```

Comment:
A nuance here is that we explicitly allow the types to be used down to 4.8, but the template comes with 5.0 for typechecking improvements with ESModule resolution.

Reply:
Great point, so something like

"Template now uses TypeScript 5.0. React Native types continue working on TypeScript 4.8"